### PR TITLE
Feature/add material

### PR DESCRIPTION
**Pull request by ElaineHuang**
import normalize.css
import font awesome css

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,14 +63,17 @@
...
     "react-redux": "^4.4.5",
     "react-router": "^2.6.1",
     "react-router-redux": "^4.0.5",
+    "react-tap-event-plugin": "^1.0.0",
     "redux": "^3.5.2",
     "redux-saga": "^0.11.0",
-    "redux-thunk": "^2.1.0"
+    "redux-thunk": "^2.1.0",  
+    "normalize.css": "^4.2.0",
+    "react-tap-event-plugin": "^1.0.0"
```
Comment (ben196888) [inline package.json]: pkg重複了。

File: package.json [hunk 2]
```diff
@@ -63,14 +63,17 @@
...
     "imports-loader": "^0.6.5",
     "normalize.css": "^4.2.0",
-    "react": "^15.2.1",
-    "react-dom": "^15.2.1",
+    "react": "^15.3.0",
+    "react-dom": "^15.3.0",
     "react-redux": "^4.4.5",
     "react-router": "^2.6.1",
     "react-router-redux": "^4.0.5",
+    "react-tap-event-plugin": "^1.0.0",
```
Comment (ben196888) [inline package.json]: https://github.com/zilverline/react-tap-event-plugin

For touch devices. You've probably heard of iOS's dreaded 300ms tap delay. React's onClick attribute falls prey to it. Facebook's working on a solution in the form of TapEventPlugin, but it won't be made available until 1.0.
https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away

File: package.json (hunk 1 above)
Comment (ElaineHuang) [inline package.json]: 哪一個哪一個?~

File: app/components/header/header.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ export default class Header extends React.Component {
       <div className="layout-row vertical-top castle-background">
         <ul className="layout-row align-between">
           <li><a href="">Flea Market</a></li>
-          <li><a href="">Logout</a></li>
+          <li><a href=""><span>Logout</span><i className="fa fa-sign-out" /></a></li>
```
Comment (ElaineHuang) [inline app/components/header/header.js]: 好的~!

File: package.json [hunk 4]
```diff
@@ -63,14 +63,16 @@
...
     "redux": "^3.5.2",
     "redux-saga": "^0.11.0",
-    "redux-thunk": "^2.1.0"
+    "redux-thunk": "^2.1.0",  
+    "normalize.css": "^4.2.0"
```
Comment (ben196888) [inline package.json]: 這也重複啦XD

File: package.json [hunk 5]
```diff
@@ -63,11 +63,12 @@
     "immutable": "^3.8.1",
     "imports-loader": "^0.6.5",
     "normalize.css": "^4.2.0",
```
Comment (ben196888) [inline package.json]: 你要的在這裡

**Pull request merged**